fire/SFMainMod.F90

```diff
@@ -902,28 +911,56 @@ subroutine  crown_damage ( currentSite )
 
           do while(associated(currentCohort))  
              currentCohort%fraction_crown_burned = 0.0_r8
+
+             
              if (EDPftvarcon_inst%woody(currentCohort%pft) == 1) then !trees only
-                ! Flames lower than bottom of canopy. 
-                ! c%hite is height of cohort
-                if (currentPatch%SH < (currentCohort%hite-currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft))) then 
-                   currentCohort%fraction_crown_burned = 0.0_r8
-                else
-                   ! Flames part of way up canopy. 
-                   ! Equation 17 in Thonicke et al. 2010. 
-                   ! flames over bottom of canopy but not over top.
-                   if ((currentCohort%hite > 0.0_r8).and.(currentPatch%SH >=  &
-                        (currentCohort%hite-currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft)))) then 
-
-                           currentCohort%fraction_crown_burned =  (currentPatch%SH-currentCohort%hite*(1.0_r8- &
-                                EDPftvarcon_inst%crown(currentCohort%pft)))/(currentCohort%hite* &
-                                EDPftvarcon_inst%crown(currentCohort%pft)) 
-
-                   else 
-                      ! Flames over top of canopy. 
-                      currentCohort%fraction_crown_burned =  1.0_r8 
-                   endif
+                
+                ! height_cbb = clear branch bole height at base of crown (m)
+                ! inst%crown = crown_depth_frac (PFT)
+                crown_depth                   = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft) 
+                height_cbb                    = currentCohort%hite - crown_depth
+                passive_crown_FI              = 0.0_r8  !critical fire intensity for passive crown fire
+                ignite_crown                  = 0.0_r8  !ratio for ignition of passive crown fire,EQ 14 Bessie & Johnson 1995
+                currentCohort%crown_fire_flg  = 0       !flag for passvie crown fire ignition
```

Comment:
as far as I can tell, crown_fire_flg is only used inside this cohort loop. 
Can we turn this into a local variable? Also, I would recommend using a logical, and also having the name passive in there somewhere.

Reply:
@rgknox updated this flag to logical. I think when I wrote this originally, I left it at the cohort level so we can collect history output on this (passive or active crown fire mortality cause). But I never added that history output. Might check in with you on that history logic/format.

Reply:
@rgknox per our email yesterday, this was reverted to binary (1 = true).